global.variables.yml

```diff
@@ -15,7 +15,7 @@ variables:
   ######################################
 
   templateSpecsDoPublish: true # Set to true, if you would like to publish module templates as template specs
-  templateSpecsRGName: 'artifacts-rg' # The name of the resource group to publish to. If the resource group does not exist, it will be created.
+  templateSpecsRGName: 'templates-rg' # The name of the resource group to publish to. If the resource group does not exist, it will be created.
```

Comment:
```suggestion
  templateSpecsRGName: 'artifacts-rg' # The name of the resource group to publish to. If the resource group does not exist, it will be created.
```